### PR TITLE
Get simulation running

### LINKED ISSUE #35: Get the simulation going
Get the field in the sim, update `physics.py`, make simulation great again.

### PR DESCRIPTION
**Pull request by james-ward**
Closes #35

### CODE REVIEW

File: physics.py [hunk 1]
```diff
@@ -4,50 +4,29 @@
...
     from robot import MyRobot
 
 
 class SimpleTalonFXMotorSim:
-    def __init__(
-        self, motor: phoenix6.hardware.TalonFX, kV: float, rev_per_unit: float
-    ) -> None:
+    def __init__(self, motor: phoenix6.hardware.TalonFX, kV: float) -> None:
         self.sim_state = motor.sim_state
+        self.sim_state.set_supply_voltage(12.0)
```
Comment (auscompgeek) [inline physics.py:22]: holy crap I can't believe it was _this_ simple, how did I miss this

File: physics.py [hunk 2]
```diff
@@ -61,16 +40,14 @@ def __init__(self, physics_controller: PhysicsInterface, robot: MyRobot):
...
+                kV=0.1,  # TODO: get from sysid logs
             )
             for module in robot.chassis.modules
         ]
         self.steer = [
             SimpleTalonFXMotorSim(
                 module.steer,
-                kV=1,  # TODO: get from sysid logs
-                rev_per_unit=1 / module.STEER_MOTOR_REV_TO_RAD,
+                kV=0.01,  # TODO: get from sysid logs
```
Comment (auscompgeek) [inline physics.py:50]: Could we use a tool such as [ReCalc](https://www.reca.lc) to better guess kV?

File: physics.py (hunk 1 above)
Comment (james-ward) [reply physics.py:22]: Yeah, I thought you'd hate this!

**Review by LucienMorey — APPROVED** [review]
LGTM. given that this works as a base we can start here and then tune robot gains as we go

**Pull request merged**